scripts/demo-native

```diff
@@ -7,7 +7,7 @@ if [ -z "${IN_NIX_SHELL-}" ]; then
     REPO_ROOT="$(dirname "$(dirname "$(readlink -fm "$0")")")"
     # Default to CARGO_TARGET_DIR if set, otherwise use the default target directory.
     TARGET_DIR="${CARGO_TARGET_DIR:-${REPO_ROOT}/target}"
-    export "PATH=${TARGET_DIR}/release:$PATH"
+    export "PATH=${TARGET_DIR}/debug:$PATH"
```

Comment:
Hmm I think this will confuse people who have been building with release. I think we should log something prominent to the terminal about this change.

Reply:
e7ce1264aee825b9c13c63e1021ef520790389f1